api/query/cache/service/main.go

```diff
@@ -200,6 +200,14 @@ func searchHandler(w http.ResponseWriter, r *http.Request) {
 	if len(missing) != 0 {
 		resp.IgnoredRuns = missing
 	}
+
+	if _, showMetadata := urlQuery[shared.MetadataFlag]; showMetadata {
+		var netClient = &http.Client{
+			Timeout: time.Second * 5,
+		}
+		resp.MetadataResult = shared.GetMetadataResponse(runs, netClient, log.StandardLogger())
```

Comment:
Generally, we get the logger from the context. This lets tests drop logs without issue.

Reply:
Again, this is on AppEngine Flex. The background context doesn't actually have a logger associated with it, so all you'll get is "Context without logger; logs will be dropped". We do need to use `log.StandardLogger()` here. And it is true that tests will print out a lot of stuff into stdout (which is already happening in search cache & revision announcer), but we should do a separate refactor to address that.

Sorry for the back and forth, Kyle.

Reply:
yea No worries!